guides/common/modules/proc_rex-pull-based-provider.adoc

```diff
@@ -0,0 +1,31 @@
+[id="REX_pull_based_provider_{context}"]
+= REX pull based provider 
+
+Remote Execution has been around for a while now and has served SSH-based execution very well.
+ 
+Now, {Project} has a ‘pull’ based provider that has the features of REX but with a deployment model similar to katello-agent. 
+
+This provider uses MQTT as its messaging protocol and includes an MQTT client running on each content host.
+
+In order to tune the MQTT server (`mosquitto`) and increase the number of clients (content hosts) connnected to it, you can follow the following procedure on the {Project} server:
+
+.Procedure
+. Enable the feature using: `{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt` (this will change the REX transport mode from the default SSH one to MQTT).
+  It's important to mention that one {Project} server ({ProjectServer} or {SmartProxyServer}) can only use one transport mode: SSH or MQTT.
```

Comment:
```suggestion
. Enable pull-based remote execution on your {ProjectServer} or {SmartProxyServer}:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt
----
+
Note that your {ProjectServer} or {SmartProxyServer} can only use one transport mode, either SSH or MQTT.
```

Maybe reword slightly here and there; but in general, we use code blocks instead of inline commands.